packages/dds/tree/src/test/simple-tree/toStoredSchema.spec.ts

```diff
@@ -133,36 +151,41 @@ describe("toStoredSchema", () => {
 							},
 							() => true,
 						);
+						// Similar to above, we check the produced restrictive schema actually behaves correctly for the test tree.
 						assert.equal(
 							restrictiveOutOfSchema === true,
 							testCase.requiresStagedSchema === true ||
 								testCase.hasUnknownOptionalFields === true,
 						);
 
 						// These arn't the tests for "exportSimpleSchema", but toStored should work with them, so we can use them to check consistency and round trip.
```

Comment:
While we're here (if you don't mind)
```suggestion
						// These aren't the tests for "exportSimpleSchema", but toStored should work with them, so we can use them to check consistency and round trip.
```